superset/assets/package.json

```diff
@@ -146,14 +146,14 @@
     "@types/react-dom": "^16.0.9",
     "babel-cli": "^6.14.0",
     "babel-core": "^6.10.4",
-    "babel-eslint": "^8.2.2",
+    "babel-eslint": "7.2.3",
```

Comment:
Why downgrade this one?

Reply:
I forgot the details but I think this was recommended on one of the thread I read `babel-eslint` 8 had deps no some beta package. Let me try to upgrade to latest instead (didn't think it was an option when I downgraded).

Reply:
Upgrading worked!